src/components/AvatarImage/__snapshots__/AvatarImage.spec.tsx.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This is being automatically generated. If I remove it from the PR, the tests fail. 🤷